src/styles/components/buttons/variables.less

```diff
@@ -0,0 +1,11 @@
+/* -----------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+banner-plugin/variables.less
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+----------------------------------------------------------------------------- */
+
+
+
+@buttons-enabled:                                                               true;
```

Comment:
We already have this set to true in `styles/vendors/cnvs/components/button`.  Thoughts on whether we repeat here for components of the same name?